packages/nuxi/src/utils/prepare.ts

```diff
@@ -68,6 +69,13 @@ export const writeTypes = async (nuxt: Nuxt) => {
     }
   }
 
+  for (const alias in nuxt.options.alias) {
+    if (aliases[alias].startsWith(nuxt.options.srcDir)) {
+      continue
+    }
+    tsConfig.include.push(aliases[alias])
+  }
```

Comment:
let's move this up into the previous for/in loop so we don't iterate twice over the aliases. We might also want to consider having different handling for directories (maybe need glob)?

Reply:
moved it up and aligned with previous alias handling in the for loop.
If it's a directory is it necessary to also add the path with `/*`?